BulletinBoard/BBBulletin.h

```diff
@@ -25,22 +25,23 @@ typedef NS_ENUM(NSInteger, BBAttachmentMetadataType) {
 @property (nonatomic, copy) NSString *title;
 @property (nonatomic, copy) NSString *subtitle;
 @property (nonatomic, copy) NSString *message;
+@property (nonatomic, retain) BBContent * content;
```

Comment:
nitpick: space between `*` and variable name
```suggestion objc
@property (nonatomic, retain) BBContent *content;
```